shotover/src/hot_reload/server.rs

```diff
@@ -95,9 +148,17 @@ impl Drop for UnixSocketServer {
         }
     }
 }
-pub fn start_hot_reload_server(socket_path: String) {
+pub fn start_hot_reload_server(socket_path: String, sources: &[Source]) {
+    let source_handles: Vec<SourceHandle> = sources
+        .iter()
+        .map(|x| SourceHandle {
```

Comment:
huh, looking at this, maybe we should combine the SourceHandle and Source types?

Maybe rename `Source` to `SourceHandle` and then we can clone the SourceHandle here instead of constructing a different type?